Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -263,7 +263,8 @@ class TabTrayViewController: UIViewController,
         case .privateTabs:
             return TabDisplayViewController(isPrivateMode: true)
         case .syncedTabs:
-            let panel = RemoteTabsPanel(state: RemoteTabsPanelState.emptyState())
+            let state = RemoteTabsPanelState.emptyState() // Temporary
```

Comment:
Similarly, for the temporary comment can you add a TODO with the ticket number it's to be updated with.